src/routes/index.js

```diff
@@ -21,9 +21,9 @@ module.exports = app => {
 
   //items
   app.post('/seller/:sellerId/items', verifyToken, Items.create);
-  app.get('/items', verifyToken, Items.list);
+  app.get('/items', Items.list);
   app.get('/seller/:sellerId/items', verifyToken, Items.getItemBySellerId); // TODO: [STYLE] Can this be changed to only a filter (query param) on the index call?
```

Comment:
@vraravam This is the api (/seller/:sellerId/items) we are using for home page. we are not using both of the get apis for which you removed the `verifyToken`.